configure.ml

```diff
@@ -511,6 +514,31 @@ let camltag = match caml_version_list with
   | x::y::_ -> "OCAML"^x^y
   | _ -> assert false
 
+(** Explanation of disabled warnings in -warn-more mode:
```

Comment:
Does the `-warn-more` mode exist or are you talking about the state where `-no-warn-error` option is not passed?